### DISCUSSION TITLE
16 Bit Image

### DISCUSSION THREAD

**Discussion by Yousaf93** [Bodmer/TFT_eSPI#2500] [Q&A - General] [answered]
Hi, I have a working animation code of using 16bit conversion of image.
My ESP32 boards version is 1.0.6
I have everything working good on windows in this settings.
I uploaded same code using MACOS with ESP32 boards latest version and same hardware but the TFT does not update any image. All other examples work fine.
Can anyone share any possible reason ?

**Comment by Bodmer** [comment, marked as answer]
This is likely to be a board package issue. Report issue here:
https://github.com/espressif/arduino-esp32

**Comment by Yousaf93** [comment]
It was board package issue. Worked with 1.0.6.
Thanks Bodmer
